From: =?utf-8?q?IOhannes_m_zm=C3=B6lnig?= <umlaeute@debian.org>
Date: Tue, 29 Jan 2019 11:31:36 +0100
Subject: rename "pd" to "puredata"

in order to allow multiple flavours of Pd

to be installed simultaneously, puredata will install itself as
"/usr/bin/puredata" and provide an alternative as "pd"

This patch ensures that the "puredata" namespace is used throughout
(both installed binary and library-paths)
---
 configure.ac | 2 +-
 src/s_main.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- puredata.orig/configure.ac
+++ puredata/configure.ac
@@ -2,7 +2,7 @@
 ##### Prelude #####
 
 AC_PREREQ(2.59)
-AC_INIT([pd], [0.51.3])
+AC_INIT([puredata], [0.51.3])
 AC_CONFIG_SRCDIR(src/m_pd.c)
 AC_CONFIG_AUX_DIR([m4/config])
 AC_CONFIG_MACRO_DIR([m4/generated])
--- puredata.orig/src/s_main.c
+++ puredata/src/s_main.c
@@ -635,7 +635,7 @@
 #else
     strncpy(sbuf, sbuf2, MAXPDSTRING-30);
     sbuf[MAXPDSTRING-30] = 0;
-    strcat(sbuf, "/lib/pd");
+    strcat(sbuf, "/lib/puredata");
     if (stat(sbuf, &statbuf) >= 0)
     {
             /* complicated layout: lib dir is the one we just stat-ed above */
